rules/linux/macos_clear_system_logs.yml

```diff
@@ -0,0 +1,24 @@
+title: Indicator Removal on Host - Clear Mac System Logs
+id: acf61bd8-d814-4272-81f0-a7a269aa69aa
+status: experimental
+description: Detects deletion of local audit logs
+author: remotephone, oscd.community
+date: 2020/10/11
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1070.002/T1070.002.md
+logsource:
+    product: macos
```

Comment:
could you please add `category: process_creation` here?

Reply:
Added in latest commit, thanks!